docs/INSTALLATION.md

```diff
@@ -0,0 +1,31 @@
+## Installing Nim
+
+### Homebrew on Mac OS X
+
+```bash
+brew update
+brew install nim
+```
+
+### Arch Linux
+
+```bash
+pacman -S nim
+```
+
+### Void Linux
+
+```bash
+xbps-install -S nim
+```
+
+### FreeBSD
+
+```bash
+pkg install nim
+```
+
```

Comment:
Might be worth mentioning choosenim as well :)

Reply:
Sure, I added text for `choosenim`. 👍